app/src/main/java/com/kamron/pogoiv/MainActivity.java

```diff
@@ -116,6 +118,7 @@
     private int pokemonHP;
     private boolean pokeFlyRunning = false;
     private int trainerLevel;
+    private LruCache<String, String> OCRCache = new LruCache<>(200);
```

Comment:
Well, I won't want to hash them all together. If the same HP occurs in another pokemon, it might still get a cache hit.

But agree that using strings isn't the best way to add a "tag" to a hash.